src/function/chatPostMessage.js

```diff
@@ -0,0 +1,72 @@
+import https from "https";
+import * as dotenv from "dotenv";
+
+dotenv.config();
+
+const TOKEN = process.env.LINE_ACCESS_TOKEN;
+
+// Request header
+const HEADERS = {
+	"Content-Type": "application/json",
+	Authorization: `Bearer ${TOKEN}`,
+};
+
+// Send four-day weather forecast information to user
+const chatPostMessage = function (replyToken, weatherForecastData, messageText) {
```

Comment:
What is your reason to choose Anonymous Function here?

Reply:
I didn't understand the use of Anonymous Function, but now I understand it and changed it to named function.